s2s/traintest.py

```diff
@@ -0,0 +1,62 @@
+"""AI4S2S train/test splitting methods.
+
+A collection of train/test splitting approaches used for cross-validation.
+"""
+import numpy as np
+import pandas as pd
+from sklearn.model_selection import KFold
+
+
+def kfold(intervals_df: pd.DataFrame, n_splits: int = 2, **kwargs):
+    """
+    K-Folds cross-validator, which splits provided intervals into k
+    consecutive folds.
+
+    For more information
+
+    args:
+        intervals_df: DataFrame to store the train/test groups.
```

Comment:
```suggestion
        df: DataFrame to store the train/test groups. It must have a column named "anchor_years"
```

1. "intervals_df" is mostly meaningful in the context of our Calendar object, whereas this function is much more generic, so I would use a more generic term.
2. We could consider making the column name "anchor_year" also settable by the user, so they could reuse the function independently from the calendar. 
3. Ideally, I would want anchor_year (or whatever other thing) to be the index. In that case, this function reduces to mostly bare sklearn, which is good. I'll see if I can come up with a proposal for that.

Reply:
Thanks for the suggestions. I totally agree! And by having the `anchor_year` as the index and attach train/test group information to it, then we can extract the train and test group separately by simply calling `pandas.DataFrame.groupby()`.

Reply:
The docstrings have been updated.